experiment.config

```diff
@@ -6,23 +6,26 @@
 ** params.sample_sheet              directory path to JSON samplesheet file (required)
 ** params.genomes_json              directory path to JSON genome file that has paths to required genome-related files (required)
 **
-** params.bcl2fastq_cpus            number of threads used by bcl2fastq (optional)
-** params.max_mem_bcl2fastq         maximum amount of memory used by bcl2fastq (optional)
-** params.demux_buffer_blocks       number of 8K byte blocks to use for demux output buffer (optiona)
+** params.bcl2fastq_cpus            number of threads used by bcl2fastq (optional, default is 6)
+** params.max_mem_bcl2fastq         maximum amount of memory used by bcl2fastq (optional, default is 40 GB total)
```

Comment:
I imagine you did some tests to land at 40GB? (If we drop to 32G might we be more likely to find available nodes, but I do not know offhand the distribution of resources on the server.)